bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -401,7 +401,9 @@ spec:
           - use
         serviceAccountName: windows-machine-config-operator
       deployments:
-      - name: windows-machine-config-operator
+      - label:
+          name: windows-machine-config-operator
```

Comment:
We've generally been [dropping this change](https://github.com/openshift/windows-machine-config-operator/pull/2382#discussion_r1720120162) even though it's generated. Still not sure why though

Reply:
right, I left it following recent [conclusion](https://github.com/openshift/windows-machine-config-operator/pull/2382#discussion_r1720163455)  😄

Reply:
that conclusion is for a different change (the operator-sdk version)

Reply:
perhaps @mansikulkarni96 can weigh in

Reply:
I am in favor of the change itself but this is something that we need to do across all branches. So should we create a separate PR and get it backported for tracking purposes?

Reply:
Makes sense, will drop it.

Reply:
Adjusted.  @mansikulkarni96  PTAL
https://github.com/openshift/windows-machine-config-operator/compare/ed10484c7a1ef4b9546af1099ba91baf6b018678..8ddb919537275c12b6f60871b41f10f5a755883a